worker/task.go

```diff
@@ -2234,6 +2250,15 @@ func (qs *queryState) evaluate(cp countParams, out *pb.Result) error {
 	for itr.Seek(countKey); itr.Valid(); itr.Next() {
 		item := itr.Item()
 		var key []byte
+		key = item.KeyCopy(key)
+		k, err := x.Parse(key)
+		if err != nil {
+			return err
+		}
+		if cp.fn == between && int64(k.Count) > counth {
+			break
```

Comment:
I thought we will continue here to the next item and not break, no?

Reply:
We need to break here as soon as we cross the upper bound in case of between.

Reply:
Is it because all future keys count will be higher. (i.e keys are sorted?)

Reply:
yes.